sqlmesh/dbt/loader.py

```diff
@@ -167,7 +167,7 @@ def _load_projects(self) -> t.List[Project]:
 
                 self._projects.append(project)
 
-                if project.context.target.database != self._context.default_catalog:
+                if project.context.target.database != (self._context.default_catalog or ""):
```

Comment:
What does an empty catalog imply?

Reply:
Clickhouse doesn't have catalogs (just 2-level object naming), so the catalog should always be empty